cos_alerter/server.py

```diff
@@ -16,6 +17,14 @@
 logger = logging.getLogger(__name__)
 
 
+def get_name_for_uuid(target_uuid, clients):
+    """Get the name corresponding to a given UUID from a list of clients."""
+    for client in clients:
+        if client["id"] == target_uuid:
+            return client["name"]
+    return None
```

Comment:
This feels inefficient. Perhaps instead of a list of client dicts, it could be a dict of the form `{clientid: client_dict}`

Reply:
I'm not clear on the details, but from the url `?clientid=<clientid>&key=<clientkey>` it seems like the basic collection is a 2-tuple: `Tuple[str, str]`.

Reply:
Currently in the config file we have `clients: [client_dict_0, ...]`. I am recommending `clients: {clientid_0: client_dict_0}`